pandas/core/categorical.py

```diff
@@ -1873,59 +1873,3 @@ def _convert_to_list_like(list_like):
     else:
         # is this reached?
         return [list_like]
-
-
```

Comment:
IIRC there are also some tests in ``test_common.py`` that go along with some of the things you moved. can you move to ``pandas/tests/types/test_concat.py`` (if they were originally in test_common; other tests you have added below are fine)